db/redis.go

```diff
@@ -0,0 +1,66 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+	"github.com/stakwork/sphinx-tribes/utils"
+)
+
+var ctx = context.Background()
+var RedisClient *redis.Client
+
+func InitRedis() {
+	redisURL := os.Getenv("REDIS_URL")
+
+	if redisURL == "" {
+		dbInt, _ := utils.ConvertStringToInt(os.Getenv("REDIS_DB"))
+		RedisClient = redis.NewClient(&redis.Options{
+			Addr:     os.Getenv("REDIS_HOST"),
+			Username: os.Getenv("REDIS_USER"),
+			Password: os.Getenv("REDIS_PASS"),
+			DB:       dbInt,
+		})
+	} else {
+		opt, err := redis.ParseURL(redisURL)
+		if err != nil {
+			fmt.Println("REDIS URL CONNECTION ERROR ===", err)
+		}
+
+		RedisClient = redis.NewClient(opt)
+	}
+}
+
+func SetValue(key string, value interface{}) {
+	err := RedisClient.Set(ctx, key, value, 6*time.Hour).Err()
```

Comment:
can we make `6*time.Hour` a var so it is more obvious incase we want to change that value in the future?